Plugin: DickyPK

**插件名称：**

DickyPK

**插件功能：**

移植自OPQBot的群聊小游戏 CHINCHIN-PK

**PyPI 项目名：**

nonebot-plugin-dicky-pk

**插件 import 包名：**

nonebot_plugin_dicky_pk

**插件项目仓库/主页链接：**

https://github.com/tkgs0/nonebot-plugin-dicky-pk

**标签：**

[{"label":"a:onebot11","color":"#000000"},{"label":"t:群聊小游戏","color":"#ffd500"}]

# 📃 商店发布检查结果

> Plugin: DickyPK

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-dicky-pk 的信息如下：
    name         : nonebot-plugin-dicky-pk 
     version      : 0.1.0                   
     description  : Dicky-PK in NoneBot2    
    
    dependencies
     - arrow &gt;=1.2.3
     - dbutils &gt;=3.0.2
     - nonebot-adapter-onebot &gt;=2.2.0,&lt;3.0.0
     - nonebot2 &gt;=2.0.0-rc.2,&lt;3.0.0
插件 nonebot_plugin_dicky_pk 加载出错：
02-04 11:46:04 [SUCCESS] nonebot | NoneBot is initializing...
02-04 11:46:04 [INFO] nonebot | Current Env: prod
02-04 11:46:04 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_dicky_pk&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_dicky_pk&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_dicky_pk/__init__.py&quot;, line 17, in &lt;module&gt;
    from .src.main import KEYWORDS, message_processor as chinchin
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_dicky_pk/src/main.py&quot;, line 1, in &lt;module&gt;
    from .db import DB, lazy_init_database
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_dicky_pk/src/db.py&quot;, line 3, in &lt;module&gt;
    from .config import Config
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_dicky_pk/src/config.py&quot;, line 16, in &lt;module&gt;
    json.dumps(
  File &quot;/usr/lib/python3.10/json/__init__.py&quot;, line 234, in dumps
    return cls(
TypeError: JSONEncoder.__init__() got an unexpected keyword argument &#x27;escape_forward_slashes&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: a:onebot11-#000000, t:群聊小游戏-#ffd500。</li><li>✅ 项目 <a href="https://github.com/tkgs0/nonebot-plugin-dicky-pk">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-dicky-pk/">nonebot-plugin-dicky-pk</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4170,5 +4170,24 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_dicky_pk",
+    "project_link": "nonebot-plugin-dicky-pk",
+    "name": "DickyPK",
+    "desc": "移植自OPQBot的群聊小游戏 CHINCHIN-PK",
+    "author": "tkgs0",
+    "homepage": "https://github.com/tkgs0/nonebot-plugin-dicky-pk",
+    "tags": [
+      {
+        "label": "a:onebot11",
+        "color": "#000000"
+      },
+      {
+        "label": "t:群聊小游戏",
+        "color": "#ffd500"
+      }
+    ],
+    "is_official": false
   }
 ]

